### PR TITLE
Refactor agencies.js names

### PR DESCRIPTION
**Pull request by pkarman**
* Include department and agency abbreviations in the search-able name.
* Pull build code into its own script with its own make target.

MERGE AFTER #82

### CODE REVIEW

File: prototypes/form-wizard/Makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+agencies:
+	ruby bin/build-agencies-js.rb
```
Comment (adborden) [inline prototypes/form-wizard/Makefile]: is bundle exec needed?

File: prototypes/form-wizard/bin/build-agencies-js.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
```
Comment (adborden) [inline prototypes/form-wizard/bin/build-agencies-js.rb:1]: does this replace the need for `_plugins/jekyll-agency-model-builder.rb`?

**Review by adborden — APPROVED** [review]

File: prototypes/form-wizard/bin/build-agencies-js.rb (hunk 2 above)
Comment (pkarman) [reply prototypes/form-wizard/bin/build-agencies-js.rb:1]: it does, and I will remove.

File: prototypes/form-wizard/Makefile (hunk 1 above)
Comment (pkarman) [reply prototypes/form-wizard/Makefile]: probably, I will add.

**Pull request merged**